docs/openapi_3.md

```diff
@@ -0,0 +1,113 @@
+# OpenAPI 3.0/3.1 Support
+
+## Quick Start
+
+```ruby
+# Swagger 2.0 (default, unchanged)
```

Comment:
letting it as default, no option adding for existend apps, is understandable,
but maybe adding one breaking change: that this must be added → UPGRADE entry
and set one of the 3.x to the default?